migrations/V152__Update_Possible_Operational_Solution_Contacts.sql

```diff
@@ -0,0 +1,34 @@
+-- Remove old entries from Solution Contacts
+DELETE FROM possible_operational_solution_contact
+WHERE NAME IN ('Felicia Addai', 'Miyani `Treva`');

```

Comment:
```suggest
WHERE NAME IN ('Felicia Addai', 'Miyani Treva');
```

The current SQL doesn't remove `Miyani Treva`

Reply:
I must have accidentally hit the backtick key on their name when copying it for PR writeup or when searching through other migrations. Fixed.